lib/auth/init.go

```diff
@@ -472,8 +476,8 @@ func ReadIdentityFromKeyPair(keyBytes, certBytes []byte) (*Identity, error) {
 	if err != nil {
 		return nil, trace.BadParameter("unsupported private key: %v", err)
 	}
-	if len(cert.ValidPrincipals) != 1 {
-		return nil, trace.BadParameter("valid principals: need exactly 1 valid principal: host uuid")
+	if len(cert.ValidPrincipals) != 2 {
```

Comment:
you can have more than two, also this makes existing nodes backwards incompatible, so you  have to regenerate host cert if you detect this situation here